### DISCUSSION TITLE
tgdownloaderblazor

### DISCUSSION THREAD

**Discussion by frezeen** [DamianMorozov/OpenTgResearcher#55] [General]
sorry stupid question, how can i use it?

is it a tgdownloader webserver version?

i click .exe but nothing happen

**Comment by DamianMorozov** [comment]
A tutorial on how to run Blazor app over Docker is in the process of being written. Wait a couple of days.

**Comment by DamianMorozov** [comment]
Check this link.
https://hub.docker.com/r/damianmorozov/tgdownloader-blazor

**Comment by DamianMorozov** [comment]
No time for the web version.
